### PR TITLE
Bxmsprod 898 - add JS Doc to functions 

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "github-ci-bot",
+  "name": "kie-ci-bot",
```
Comment (Ginxo) [inline package.json:2]: this shouldn't be in this PR but in https://github.com/kiegroup/github-ci-bot/pull/5

File: src/lib/ci.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 const globToRegExp = require('glob-to-regexp');
 const { getChangedFiles } = require("./utils");
 
-
+//isCIRequired checks if we need to run the Jenkins job(based to trigger only on comment) and comment the required phrase, which is defined in comments.yml
```
Comment (Ginxo) [inline src/lib/ci.js:4]: please use [JSDoc](https://jsdoc.app) instead.
something like
```
/**
 * checks if we need to run the Jenkins job(based to trigger only on comment) and comment the required phrase, which is defined in comments.yml
 * @param {string} context - The context from which the PR is coming from
 * @returns {boolean} true/false depending on the method logic
 */
```

Comment (Ginxo) [reply src/lib/ci.js:4]: the same for the rest of the methods (just the ones from `src/lib` folder)

Comment (Kaitou786) [reply src/lib/ci.js:4]: Okay changing.

File: package.json (hunk 1 above)
Comment (Kaitou786) [reply package.json:2]: Yeah, I sent this PR over that one so no merge conflict will happen. will remove this commit.

File: src/lib/ci.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 const globToRegExp = require('glob-to-regexp');
 const { getChangedFiles } = require('./utils');
 
+/**
+ * checks if we need to run the Jenkins job(based to trigger only on comment) and comment the required phrase, which is defined in comments.yml
+ * @param {string} context - The context from which the PR is coming from
+ * @returns {boolean} true/false depending on the method logic
+ */
+
```
Comment (Ginxo) [inline src/lib/ci.js:9]: remove blank line

File: src/lib/review.js [hunk 4]
```diff
@@ -9,7 +13,12 @@ async function askToReview(context) {
     })
   );
 }
-
+/**
+ * Calculates all the required reviewers needed for the PR and return them as an Array
+ * @param {string} context The context from which the PR is coming from
+ * @param {yaml} reviewersInfo The parsed yaml defined in reviewers.yml
```
Comment (Ginxo) [inline src/lib/review.js:19]: `yaml` is not supported by JSDoc https://jsdoc.app/tags-type.html
I would use just `Object` or `Object.<string, Object>` the `yaml` part is already defined in the param description, which is clear enough

Comment (Ginxo) [reply src/lib/review.js:19]: the same for label.getRequiredLables

**Pull request merged**